packages/server/src/fhir/repo.ts

```diff
@@ -1830,6 +1830,15 @@ export class Repository extends FhirRepository<PoolClient> implements Disposable
   private async getAccounts(existing: Resource | undefined, updated: Resource): Promise<Reference[] | undefined> {
     const updatedAccounts = this.extractAccountReferences(updated.meta);
     if (updatedAccounts && this.canWriteAccount()) {
+      if (updated.resourceType !== 'Patient') {
+        // Log intentional writes to accounts on non-Patient resources
+        if (
+          !existing ||
+          new Set(this.extractAccountReferences(existing.meta)).difference(new Set(updatedAccounts)).size > 0
```

Comment:
Actually now that I'm checking this, this only fires when the first set contains an element that the second set does not, and not the other way around

There is also `.symmetricDifference` if we want to know if there is a missing element on either side of the comparison https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/symmetricDifference

Reply:
@ThatOneBro Great catch!

Reply:
I would have expected the default behavior to be what `.symmetricDifference` is doing, but seems these folks are big set theory people